oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/directory/ActiveDeletedBlobCollectorFactory.java

```diff
@@ -324,7 +323,11 @@ public void purgeBlobsDeleted(long before, @NotNull GarbageCollectableBlobStore
             long startBlobTrackerSyncTime = clock.getTime();
             // Synchronize deleted blob ids with the blob id tracker
             try {
-                Closeables.close(idTempDeleteWriter, true);
+                try {
```

Comment:
Can't we create an Util for this. Instead of repeating these 4 lines, put them under an Utility class in oak-commons. wdyt @reschke ?

Reply:
I really don't think we need this. In this case, inlining actually makes things clearer.